config-templates/redhat/odf-local/changelog.json

```diff
@@ -3,114 +3,105 @@
-            },
-            "previous-revision": {
-                "revision": "13",
-                "release-date": "20 February 2023",
-                "cves": "CVE-2022-47629",
-                "golang": "1.18.9",
-                "ubi": "8.7-1049.1675784874",
-                "new-features": [],
-                "fixes": []
             }
         }
     },
     {
         "4.9": {
             "status": "supported",
             "latest-revision": {
+                "revision": "21",
+                "release-date": "21 March 2023",
+                "cves": "CVE-2023-23916",
+                "golang": "1.19.6",
+                "ubi": "8.7-1085",
+                "new-features": [],
+                "fixes": []
+            },
+            "previous-revision": {
                 "revision": "20",
                 "release-date": "06 March 2023",
                 "cves": "CVE-2022-4415,CVE-2020-10735,CVE-2021-28861,CVE-2022-45061,CVE-2022-40897,CVE-2022-41723,CVE-2022-41724",
                 "golang": "1.19.6",
                 "ubi": "8.7-1085",
                 "new-features": [],
                 "fixes": ["Default values for CPU limit & Memory limit are updated to 500m & 500Mi respectively."]
-            },
-            "previous-revision": {
-                "revision": "19",
-                "release-date": "20 February 2023",
-                "cves": "CVE-2022-47629",
-                "golang": "1.18.9",
-                "ubi": "8.7-1049.1675784874",
-                "new-features": [],
-                "fixes": []
             }
         }
     },
     {
         "4.8": {
             "status": "supported",
             "latest-revision": {
+                "revision": "21",
+                "release-date": "21 March 2023",
+                "cves": "CVE-2023-23916",
+                "golang": "1.19.6",
+                "ubi": "8.7-1085",
+                "new-features": [],
+                "fixes": []
+            },
+            "previous-revision": {
                 "revision": "20",
                 "release-date": "06 March 2023",
                 "cves": "CVE-2022-4415,CVE-2020-10735,CVE-2021-28861,CVE-2022-45061,CVE-2022-40897,CVE-2022-41723,CVE-2022-41724",
                 "golang": "1.19.6",
                 "ubi": "8.7-1085",
                 "new-features": [],
                 "fixes": ["Default values for CPU limit & Memory limit are updated to 500m & 500Mi respectively."]
-            },
-            "previous-revision": {
-                "revision": "19",
-                "release-date": "20 February 2023",
-                "cves": "CVE-2022-47629",
-                "golang": "1.18.9",
-                "ubi": "8.7-1049.1675784874",
-                "new-features": [],
-                "fixes": []
             }
         }
     },
     {
         "4.7": {
-            "status": "deprecated",
-            "latest-revision": {
+            "status": "unsupported",
+            "previous-revision": {
```

Comment:
No need to have any revision info once template is unsupported
Refer this: https://github.com/IBM/ibm-satellite-storage/blob/develop/config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json#L27